src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2190,4 +2202,15 @@ PeerImp::getScore (bool haveItem)
    return score;
 }
 
+bool
+PeerImp::highLatency ()
```

Comment:
The names and implementations of both `getScore` and `highLatency` suggest that they should marked `const`.